### ISSUE TITLE
Integrate Media Library from core

### ISSUE THREAD

**Issue by kevinfunk** [acquia/df#184]
That last thing to do to integrate Media library is to update images to to the Media library widget.

**Comment by saltednut** [comment]
Is this still outstanding in any way?

**Comment by kevinfunk** [comment]
Yes, waiting on the release of 8.8 for the major changes to increase the themeability. It was fine with Claro, but I wasn't able to select images when it opened in One.
